scripts/build_coverage.sh

```diff
@@ -88,10 +93,27 @@ source "$CHIP_ROOT/scripts/activate.sh"
 
 # Generates ninja files
 if [ "$skip_gn" == false ]; then
-    gn --root="$CHIP_ROOT" gen "$OUTPUT_ROOT" --args='use_coverage=true'
+    gn --root="$CHIP_ROOT" gen "$OUTPUT_ROOT" --args='use_coverage=true chip_can_build_all_clusters_app=true'
```

Comment:
Why aren't we building the all-clusters-app as a separate build invocation? Why does it have to be bundled in here together?

Reply:
See the comments above, this is to guarantee with only profile the source file once with same compile/profile options.